src/images/linshare-desktop-icon.svg

```diff
@@ -1,79 +1,75 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 15.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="15.357px" height="23.893px" viewBox="0 0 15.357 23.893" enable-background="new 0 0 15.357 23.893" xml:space="preserve">
-<g id="Calque_1_1_" display="none">
-	<g display="inline">
-		<rect id="Calendar_3_" x="-26.365" y="0.045" fill="none" width="23.897" height="23.897"/>
-		<g>
-			<path id="calendar_3_" fill="#2196F3" d="M-9.534,17.623h-9.839V9.892h9.839 M-11.64,4.971v1.406h-5.623V4.971h-1.406v1.406
+<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="15.357px" height="23.893px" viewBox="0 0 15.357 23.893" enable-background="new 0 0 15.357 23.893" xml:space="preserve">

```

Comment:
Why did you change these svg files?

Reply:
I just reformatted them and removed some unnecessary metadata, as they are directly injected in HTML as inline SVGs now.